nmigen/lib/roundrobin.py

```diff
@@ -0,0 +1,53 @@
+from .. import *
+
+
+__all__ = ["RoundRobin"]
+
+
+class RoundRobin(Elaboratable):
+    """Round-robin scheduler.
+
+    For a given set of requests, the round-robin scheduler will
+    grant one request.
```

Comment:
I feel like this should be elaborated. Something like (phrasing needs to be improved):

> For a given set of requests, the round-robin scheduler will grant one request. Once it grants a request, if any other requests are active, it grants the next active request with a greater number, restarting from zero once it reaches the highest one. Use :class:`EnableInserter` to control when the scheduler advances.

Reply:
I tweaked a little bit what you wrote. What do you think?